docs/firstobject.md

```diff
@@ -105,11 +107,45 @@ We will come back to this later.
 </div>
 </details>
 
-Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline.
+Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline. 
 
 Note: Don't press `Save`, Don't press `Preview`. Why? You are not ready yet!
 
+### Linked Data
+
 ![Ingest Step2](../imgs/ingest-step2.jpg)
 
-This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as a open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty. More to come!
+As the name of this step suggests; on this page you will be adding all your Linked Data for your Digital Object. This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
+
+Click: `Next: Collections`
+
+### Collections
+
+![Ingest Step3](../imgs/inges-step-3_Collections.jpg)
+
+Since this is our first digital object we do not yet have a Digital Object *Collection* for which `My First Digital Object` could be a member of. In other words, you can leave `Collection Membership` blank and click `Next: Upload Files`.
+
+### Upload Files
+
+![Ingest Step4](../imgs/ingest-step-4_Upload-files.jpg)
+
+This step is pretty straight forward: Click `Choose Files` to open your computer's directory and select which file you would like to ingest. After you've done this, click `Save Metadata`
```

Comment:
I would suggest a small change here
Instead of "to open your computer's directory"
something like "to open your a file selector window" and choose.

I would also add a note that each "upload field" has a set of allowed File Types and in the demo there are also file size restrictions that are explained on the upload field's description

Reply:
Goood